PublishSignedPackages.cmd

```diff
@@ -18,17 +18,11 @@ set PACKAGE_ROOT=bld\bin\nuget\%ID%.%VERSION%
 call %NUGET% push %PACKAGE_ROOT%.nupkg -Source %SOURCE%
 if "%ERRORLEVEL%" NEQ "0" (echo push to %SOURCE% FAILED && goto Exit)
 
-call %NUGET%push %PACKAGE_ROOT%.symbols.nupkg -Source https://nuget.smbsrc.net/
```

Comment:
So, turns out the call above publishes symbols as well with latest nuget.exe, so this call is not necessary